.circleci/config.yml

```diff
@@ -94,9 +94,23 @@ jobs:
           name: "Check that there are no generated changes"
           command: make check-no-changes
 
+  deploy_test:
+    executor: ubuntu_vm
```

Comment:
Can you use the [cimg/base](https://circleci.com/developer/images/image/cimg/base) Docker executor, with `resource_class=small`, since this has low resource requirements?

Reply:
That is a Docker executor, so the script is running on Docker and does not have a docker daemon it can start containers in. There is a `setup_remote_docker` command which adds a `docker` command to the path that calls out the docker daemon running the executor, but when I tried it, it had issues with binding a mount (which I use to pass my python script into the container).

Reply:
Thanks for the explanation. Makes sense.